public/script.js

```diff
@@ -481,24 +311,29 @@ function previousDate() {
   }
 }
 
-/** Method to update the state of the loop*/
+/** Method to update the state of the loop
```

Comment:
Just as a note, `method` is usually a term applied to the behaviors encapsulated by classes. Since these are defined outside of a class, the correct term would just be 'function'. You can probably omit this language altogether though, as the `function` keyword makes it clear what the object is.

Instead, a good practice is to make the first line of your docstrings imperative: e.g., `update the state of the loop and return nothing`